monitoring/README.md

```diff
@@ -0,0 +1,57 @@
+## Indexer Monitoring Dashboard
+A monitoring dashboard displaying indexer performance metrics. 
+
+### To start a monitoring dashboard
+Configurable env vars, `DB_CONNECTION`, `ALGOD_NET` and `ALGOD_TOKEN`. The defaults are set in `.env`. User must provide a valid `ALGOD_NET` and `ALGOD_TOKEN` pair.
+
+if indexer is not already running, start all services in `docker-compose.yml`
+
+`docker-compose up -d`
+
+if indexer is already running, start monitoring services only.
+
+`docker-compose up -d prometheus grafana`
+ 
+prometheus target should be updated to listen to a correct indexer address (host:port), 
+
+```json
+static_configs:
+- targets: ['indexer:8888'] 
+```
+
+By default, 
+ - indexer is running on http://localhost:8888 
+ - grafana is running on http://localhost:3000; default login (admin/admin)
+
+### View metrics on grafana
+
+- In grafana configurations, add a PostgreSQL datasource. See example below. 
+- Go to Import and upload dashboard.json
+- Run `create extension pg_stat_statements;` sql on db to enable query stats from Postgres
```

Comment:
This isn't working for me. After importing the dashboard everything is blank:
![image](https://user-images.githubusercontent.com/125509/154365331-e40326a6-d401-4bff-975e-d6c4a3475c8f.png)

In the panels there is an error:
![image](https://user-images.githubusercontent.com/125509/154365420-8e4cdac1-2c4f-47f9-9dea-def86b619fb3.png)

Reply:
I was able to import on deployment outside of docker, but the postgres portion didn't seem to work. The query ran fine in pqsl but didn't show up in grafana.